RecoLocalCalo/HcalRecProducers/src/ZdcSimpleReconstructor.h

```diff
@@ -31,7 +32,8 @@
       DetId::Detector det_;
       int subdet_;
       HcalOtherSubdetector subdetOther_;
-      edm::EDGetTokenT<ZDCDigiCollection> tok_input_;
+      edm::EDGetTokenT<ZDCDigiCollection> tok_input_hcal;
+      edm::EDGetTokenT<ZDCDigiCollection> tok_input_castor;
```

Comment:
there is somewhat trivial difference between this and 81X: the two lines with tok_input_hcal and tok_input_castor are flipped.
Please keep a consistent backport.